flycheck.el

```diff
@@ -6900,6 +6900,22 @@ See URL `http://jade-lang.com'."
 (flycheck-def-config-file-var flycheck-jshintrc javascript-jshint ".jshintrc"
   :safe #'stringp)
 
+(flycheck-def-option-var flycheck-jshint-extract nil javascript-jshint
+  "Whether jshint should extract Javascript from HTML before linting.
+
+The value of this variable is one of 'auto', 'always' or 'never'.
+
+Refer to the jshint manual at the URL
+  `http://jshint.com/docs/cli/#flags'
+for more information."
+  :type '(choice (const :tag "No extraction rule" nil)
```

Comment:
@tealeg Sorry, did you miss my comment?  I would like to understand how this option works before I merge this pull request :relaxed: